R/data-set.R

```diff
@@ -197,6 +197,18 @@ DataSet <- R6::R6Class(
       }
     },
 
+    #' @description
+    #' Save the \code{DataSet} to a pkml file that can be loaded by MoBi
+    #' @param filePath Path to the location (incl. file name)
+    saveToPKML = function(filePath){

```

Comment:
In order to stay consistent with what we have done for other objects, shouldn't we create a utility-data-set or sthg and define this method there instead? Also this is were you could create some help methods around dataSet such as createDataSet(x=, y= etc..)

Reply:
Ok, then "saveDataSetToPKML" or "export..."? We are using "export" for simulation results etc, but "save" sounds somehow more intuitive for me..

Reply:
saveDataSetToPKML
We can also create aliases for other method (so that we are consistent and always have save..xx)

Reply:
@msevestre We would have to expose the `DataRepository` of the `DataSet` in order to save it via a utility function...

Reply:
Hontestly I do not have an issue with exposing the underlying DataREpository as public readonly {get;}. We can create the object tusign a DataRepo as parameter so why not?

Reply:
ok, I will expose it (readonly).

Reply:
any update on this? Id' like to merge asap